CompetitionCode/src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -23,34 +23,31 @@
 public class Storage extends SubsystemBase {
   // FIELDS
   private final VictorSP motor;
-  private final DigitalInput[] limitSwitches;
+  private final DigitalInput intakeSwitch, hasBallSwitch;
   private boolean isOverridden;
+  public int numBalls;
 
   public Storage() {
-    motor = new VictorSP(STORAGE_GATE_MOTOR);
+    motor = new VictorSP(STORAGE_GATE_MOTOR_PORT);
     isOverridden = false;
 
-    limitSwitches = new DigitalInput[6];
-    for (int i = 0; i <= 5; i++) {
-      limitSwitches[i] = new DigitalInput(i);
-    }
+    intakeSwitch = new DigitalInput(STORAGE_INTAKE_SWITCH_PORT);
+    hasBallSwitch = new DigitalInput(STORAGE_BALL_SWITCH_PORT);
 
   }
 
-  public boolean isSwitchPressed(final int index) {
-    return limitSwitches[index].get();
-  }
-
-  public DigitalInput getFirstLimit() {
-    return limitSwitches[0];
+  /**
+   * intakeSwitchSwitch getter method
+   */
+  public boolean isIntaking() {
```

Comment:
the name of this method is weird- it should be something along the lines of isIntakeActive or somethin